Dashboard/Views/ConnectionView.axaml

```diff
@@ -1,8 +1,64 @@
-﻿<UserControl xmlns="https://github.com/avaloniaui"
-             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-             mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
-             x:Class="Dashboard.Views.ConnectionView">
-    Connection Page Test
-</UserControl>
+﻿<UserControl  xmlns="https://github.com/avaloniaui"
+              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+              xmlns:vm="using:Dashboard.ViewModels"
+              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+              xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+              x:DataType="vm:ConnectionViewModel"
+              x:Class="Dashboard.Views.ConnectionView"
+              mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
+              Foreground="Black"
+              Background="#c4c4c4">
+    <Design.DataContext>
+        <!-- This only sets the DataContext for the previewer in an IDE,
+             to set the actual DataContext for runtime, set the DataContext property in code (look at App.axaml.cs) -->
+        <vm:ConnectionViewModel />
+    </Design.DataContext>
+    <Panel>
+        <ThemeVariantScope RequestedThemeVariant="Light">
+        <!-- Main Content Grid -->
+        <Grid RowDefinitions="Auto,Auto">
```

Comment:
Looks good :) maybe just center the inputs so its the middle of the screen, similar to status and scrutineering page

Reply:
Can do.
https://github.com/nickowen44/AV-Datalogger/pull/15/commits/0524be42bccec030cc15bde8f9ad52fcac5a4e11